inc/Remote/ApiCall.php

```diff
@@ -175,6 +178,30 @@ public function setDescription(string $description): self
         return $this;
     }
 
+    /**
+     * Returns the docblock tags that have not been processed specially
+     *
+     * @return array[]
+     */
+    public function getTags()
+    {
```

Comment:
What are these tags for?

Reply:
This gives access to all the doc block tags (like `@author` or `@see`).